src/tools/spice2xyzv/spice2xyzv.cpp

```diff
@@ -588,7 +588,11 @@ int main(int argc, char* argv[])
     }
 
     // Load the leap second kernel
+#if defined(WIN32) || defined(__APPLE__)
```

Comment:
I'm not sure. It will search for the file in a current directory.

Reply:
I have tested and it will search the file in the current directory.